### DISCUSSION TITLE
Update: Working Now - Invalid Username or password

### DISCUSSION THREAD

**Discussion by cyb3rl4d** [bubuntux/nordvpn#207] [Q&A]
Hi Everyone,

UPDATE - Its now working?!?

My container stopped working recently with the following error messages:
```
It's not you, it's us. We're having trouble reaching our servers. If the issue persists, please contact our customer support.
Invalid Username or password.
```

Nothing has changed my end (as far as im aware) other than updating to the latest version, I've followed the pinned issue with v3.10.0 but I'm still having the issue if I use an older version?

Im running Ubuntu 20.04.2 LTS with the following config:
```
version: "3"
services:
  vpn:
#    image: ghcr.io/bubuntux/nordvpn:v3.9.5-1 #same issue as with latest
    image: ghcr.io/bubuntux/nordvpn:latest
    container_name: NordVPN_Gateway
    ulimits: # set these to unlimited
        memlock:
          soft: -1
          hard: -1
    network_mode: bridge
    cap_add:
      - NET_ADMIN
#      - SYS_ADMIN    #replaced with SYS_MODULE same issue with or without
      - SYS_MODULE    #Replaced with SYS_ADMIN from https://github.com/bubuntux/nordvpn/issues/199, doent help
    sysctls:
      - net.ipv4.conf.eth0.rp_filter=2
    devices:
      - /dev/net/tun
    environment:
      - USER=[My Username Here]
      - PASS=[My Password Here]
      - CONNECT=--group p2p United_Kingdom    #changed to the new format
      - TECHNOLOGY=NordLynx
      - NETWORK=192.168.1.0/24
      - GROUPID:0   #removed to see if this helps, again it doesnt
      - TZ:Europe/London  #Added to see if this helps, again it doesnt
      - DNS:192.168.1.1  #Added to see if this solves the problem, it hasnt
      - DEBUG=Trace
    ports:
      - 1111:1111
      - 2222:2222
      - 3333:3333
      - 4444:4444
      - 5555:5555
      - 6666:6666
    restart: on-failure
```

I've reset my Nord password too just in case there is any issue there.
What am I missing, wanted to check here just in case I'm missing something from the pinned issue before I report a new issue

Thanks

**Comment by hwhite36** [comment]
I have the exact same problem but it hasn't started working again.

**Reply by cyb3rl4d** [reply]
hope its all working for you now, for me it was if their auth systems wasnt working or something

**Reply by hwhite36** [reply]
Just tried it again and it's working, I think your hypothesis about their authentication process being temporarily down is probably right. Thanks!

**Comment by GastonDiggler** [comment]
I have the same problem, if I connect with the client directly, I don't have problem.
And sometime it's work
